examples/resources/genesyscloud_webdeployments_configuration/resource.tf

```diff
@@ -75,6 +75,7 @@ resource "genesyscloud_webdeployments_configuration" "exampleConfiguration" {
     channels               = ["Webmessaging", "Voice"]
     mask_selectors         = [".my-class", "#my-id"]
     readonly_selectors     = [".my-class", "#my-id"]
+    pause_criteria         = []
```

Comment:
You can update this to 

```hcl
pause_criteria {
  url_fragment = "/sensitive"
  condition    = "includes"
}
```

Reply:
pause_criteria         = [{
                                url_fragment = "/sensitive"
                                condition    = "includes"
                              }]
probably more correct, because its list

Reply:
Nope. We actually had issues defining lists as in your example - terraform couldn't detect when a nested field was required or not. 

It is a bit strange, but that's how you define schema list items in HCL. Even if a json request body contains a field that is a map, we have to define it as a list of maps in the schema and set MaxItems: 1 to make sure the user doesn't try to define multiple.